db/sg_replicate_cfg.go

```diff
@@ -995,7 +995,9 @@ func (m *sgReplicateManager) GetReplication(replicationID string) (*ReplicationC
 	}
 
 	// TODO: remove the local/non-local handling below when CBG-909 is completed
-	if replication.AssignedNode == m.localNodeUUID {
+	if replication.AssignedNode == "" {
+		replication.AssignedNode = ""
+	} else if replication.AssignedNode == m.localNodeUUID {
```

Comment:
I'd suggest either an early return to avoid the reassignment to the same value.

```suggestion
	// unassigned replication
	if replication.AssignedNode == "" {
		return replication, nil
	}

	if replication.AssignedNode == m.localNodeUUID {
```